library/src/main/java/bio/terra/landingzone/library/LandingZoneManagerProvider.java

```diff
@@ -51,10 +47,6 @@ public AzureResourceManager createAzureResourceManagerClient(
   }
 
   public TokenCredential buildTokenCredential() {
-    return new ClientSecretCredentialBuilder()
-        .clientId(azureConfiguration.getManagedAppClientId())
-        .clientSecret(azureConfiguration.getManagedAppClientSecret())
-        .tenantId(azureConfiguration.getManagedAppTenantId())
-        .build();
+    return new DefaultAzureCredentialBuilder().build();
```

Comment:
DefaultAzureCredentialBuilder will pull in the needed credentials from the environment.